src/screens/App/Project/index.tsx

```diff
@@ -70,17 +69,14 @@ export const getServerSideProps: GetServerSideProps = async (context) => {
   // eslint-disable-next-line @typescript-eslint/no-var-requires
   const ApplicationManager = require("&server/mongodb/actions/ApplicationManager");
   // eslint-disable-next-line @typescript-eslint/no-var-requires
-  const UserManager = require("&server/mongodb/actions/UserManager");
 
   try {
-    const session = await getSession({ req: context.req as any });
+    const session = await getSession(context.req as NextApiRequest);
 
     if (session?.user == null) {
       throw new Error("User is not logged in!");
     }
 
-    const user = await UserManager.getUserById((session.user as any).id);
```

Comment:
Eliminated the need to fetch this object since the value already exists on sessions